### PR TITLE
CEDS-1092 Errors for field pairs

### PR DESCRIPTION
**Pull request by MaciejRewera**

### CODE REVIEW

File: app/forms/declaration/additionaldocuments/DocumentWriteOff.scala [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  implicit val format = Json.format[DocumentWriteOff]
+
+  private val measurementUnitMaxLength = 4
+  private val documentQuantityMaxLength = 16
+  private val documentQuantityMaxDecimalPlaces = 6
+
+  val measurementUnitKey = "measurementUnit"
+  val documentQuantityKey = "documentQuantity"
+
+  val mapping = Forms
```
Comment (PauloPortugal) [inline app/forms/declaration/additionaldocuments/DocumentWriteOff.scala]: :+1:

File: app/services/ItemsCachingService.scala [hunk 2]
```diff
@@ -90,25 +90,33 @@ class ItemsCachingService @Inject()(cacheService: CustomsCacheService)(appConfig
...
     } yield documentIdentifier + documentPart
 
+  private def createAdditionalDocumentWriteOff(doc: DocumentsProduced): Option[WriteOff] =
+    for {
+      documentWriteOff <- doc.documentWriteOff
+      measurementUnit <- documentWriteOff.measurementUnit
+      quantity <- documentWriteOff.documentQuantity
+    } yield WriteOff(quantity = Some(Measure(unitCode = Some(measurementUnit), value = Some(quantity))))
+
+//    writeOff = doc.documentQuantity.map(
```
Comment (PauloPortugal) [inline app/services/ItemsCachingService.scala]: Can you please remove these comments?

File: app/services/ItemsCachingService.scala [hunk 3]
```diff
@@ -102,6 +102,13 @@ class ItemsCachingService @Inject()(cacheService: CustomsCacheService)(appConfig
       )
     )
 
+  private def createDocumentIdentifierAndPart(doc: DocumentsProduced): Option[String] =
```
Comment (PauloPortugal) [inline app/services/ItemsCachingService.scala]: :+1:

**Comment by MaciejRewera** [comment]
Closing PR, because it was approved, but there are significant conflicts after rebasing.

**Pull request closed without merging**